### PR TITLE
Proxy setup

### PR DESCRIPTION
**Pull request by dmoygit**
PR highlights
1) added readme instructions
2) added margin so the footer bar does not cover up the page selector.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+
+For Development:
+
+1) npm install
+2) Have MySQL5.7 installed and started
+3) npm run seed (will seed the MySQL db with sample data)
+4) npm run build (to start webpack bundling)
+5) npm run start-dev (starts the server)
+
+For your Proxy Server:
```
Comment (ga-possum) [inline README.md:13]: Proxy doesn't use mysql, isn't it?

**Review by yerx — APPROVED** [review]
lgtm

**Review by ga-possum — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (dmoygit) [reply README.md:13]: no, but if you're going to use Reviews Service on your proxy server, you need the Mysql db with the service

**Pull request merged**